src/wallet/walletdb.cpp

```diff
@@ -47,6 +48,7 @@ const std::string OLD_KEY{"wkey"};
 const std::string ORDERPOSNEXT{"orderposnext"};
 const std::string POOL{"pool"};
 const std::string PURPOSE{"purpose"};
+const std::string QUEUED_NAME_TX{"queued_name_tx"};
```

Comment:
Nit: I suggest renaming this (constant and the actual string value) to just "queued tx" (without name), as it is now not tied to names at all.

Reply:
OK